content/applications/sales/sales/products_prices/prices/pricing.rst

```diff
@@ -317,6 +317,43 @@ product form.
 
 .. _sales/advanced-price-rules:
 
+Discounts
+=========
+
+The :guilabel:`Discounts` feature allows the ability to set a discount or increase the price
+on *individual items* in a sales order. This is calculated as a percentage of that product's sales
+price.
+
+To access discounts, navigate to the *Sales* app setting page
+(:menuselection:`Sales app --> Configuration --> Settings`), and click the :guilabel:`Discounts`
+checkbox, then click :guilabel:`Save`.
```

Comment:
Early line break (you can break a line inside a :menuselection: anywhere there is a space) :)

For example:
```rst
(:menuselection:❌`Sales app -->✅Configuration --> Settings`)
```


```suggestion
To access discounts, navigate to the *Sales* app setting page (:menuselection:`Sales app -->
Configuration --> Settings`), and click the :guilabel:`Discounts` checkbox, then click
:guilabel:`Save`.
```